app-shells/scsh/scsh-0.6.7-r3.ebuild

```diff
@@ -15,33 +15,42 @@ LICENSE="BSD"
 SLOT="0"
 KEYWORDS="amd64 ppc sparc x86"
 
-DEPEND="!dev-scheme/scheme48"
+DEPEND="
+	!dev-scheme/scheme48
+	virtual/libcrypt:=
+	amd64? ( virtual/libcrypt[abi_x86_32] )
```

Comment:
slot is already added. would portage error on this?

Reply:
Portage recorded the deps on my machine as `!dev-scheme/scheme48 virtual/libcrypt:0/1= virtual/libcrypt[abi_x86_32]`. Since `:0/1=` is stricter, it should eliminate others pulled in by the non-slot variant. One could assume that portage merges the USE flag in the non-slot variant with the slotted variant as well. If it were me however, I would use the slot operator on both deps to be on the safe side.

Reply:
sure